packages/api/src/routes/medical/internal-patient.ts

```diff
@@ -288,44 +287,6 @@ router.delete(
   })
 );
 
-/** ---------------------------------------------------------------------------
- * GET /internal/patient/:patientId/link
```

Comment:
Not used anymore, and replaced by the more complete `GET /internal/hie/patient/overview` endpoint